MonoZelda/Commands/GameCommands/PlayerMoveCommand.cs

```diff
@@ -22,7 +22,7 @@ public PlayerMoveCommand(Player player)
 
     public void Execute(params object[] metadata)
     {
-        Keys pressedKey = (Keys) metadata[0];
+        Keys pressedKey = (Keys)metadata[0];
```

Comment:
Why did you remove the space here? Is that standard c# notation? I'm just curious I have no clue.